### ISSUE TITLE
BUG: Filtering table view on '' no longer not works

### ISSUE THREAD

**Issue by EmanNo1** [beekeeper-studio/beekeeper-studio#2241]
**Describe the bug**
When filtering in a table view, Field = '' does not return any results

**To Reproduce**
1. Select a field name from the pull down
2. Select the type to "="
3. Enter '' as the value
4. No results are shown

**Expected behavior**
Should return rows where the select field is (Empty)

**Screenshots**
![image](https://github.com/beekeeper-studio/beekeeper-studio/assets/2874846/8496df82-785f-4920-8864-ca6bf30fe745)

After filter
![Selection_064](https://github.com/beekeeper-studio/beekeeper-studio/assets/2874846/7e481f29-312a-414e-a8ac-1812edc6eb91)

**Version Information (please complete the following information):**
 - OS: Ubuntu 24.10
 - App 4.3.4
 - Database MS SQL 2012

If I toggle the filter type and view the raw sql. I see that the '' is double escaped to ''''
`[Email] = ''''''`

![Selection_065](https://github.com/beekeeper-studio/beekeeper-studio/assets/2874846/22c0c021-9d2d-42d6-b80f-eea4bd050ebd)

If I correct the '''' to '', results are shown as expected

**Comment by rathboma** [comment]
Good find, thank you!

**Comment by rathboma** [comment]
@azmy60 possible to add a simple Playwright test for this?

**Comment by azmy60** [comment]
Yes. I feel like UI test is kinda flaky, but hopefully it'll be fine.

**Comment by QuentiumYT** [comment]
Should it act as "Emails equals empty"? why not add a `is empty` and `is not empty` from the dropdown so it's more UX?

**Comment by julianokdp** [comment]
> Good find, thank you!

It looks like this issue was forgotten. I was about to open this as a FEATURE/BUG: it’s not possible to filter by fields with empty values (EMPTY), but I noticed that this suggestion was already accepted in 2024.

Beekeeper 5.5.1
Ubuntu 24.04.2

![Image](https://github.com/user-attachments/assets/07ac414e-09e3-4c2d-9223-b146a06795ac)
